LibraBFT/Impl/Handle.agda

```diff
@@ -50,6 +50,7 @@ postulate -- TODO-2: define GenesisInfo to match implementation and write these
 initSR : SafetyRules
```

Comment:
Does `sdPreferredRound` need to be initialized too?

Reply:
> Does `sdPreferredRound` need to be initialized too?

At the moment we aren't verifying anything about `sdPreferredRound`, so its initialization is not (yet) needed. I only added initialization to be able to prove the currently tracked invariants.